salt/elife-dashboard/dashboard2.sls

```diff
@@ -0,0 +1,80 @@
+{% set app = pillar.elife_dashboard_2 %}
+{% set dash = pillar.elife_dashboard %}
+{% set user = pillar.elife.deploy_user.username %}
+
+
+#
+# db
+#
+
+
+
+#
+# configure
+#
+
+configure-{{ app.name }}:
+    file.managed:
+        - user: {{ user }}
+        - name: /srv/app/app.cfg
```

Comment:
this should be either `/srv/app/` or `/srv/elife-dashboard` (the latter is preferred, but whatever the existing one is)

Reply:
Changed to `/srv/elife-dashboard`